lti_consumer/outcomes.py

```diff
@@ -0,0 +1,205 @@
+"""
+This module adds support for the LTI Outcomes Management Service.
+
+For more details see:
+https://www.imsglobal.org/specs/ltiomv1p0
+"""
+
+import logging
+import urllib
+
+from lxml import etree
+from xml.sax.saxutils import escape
+
+from xblockutils.resources import ResourceLoader
+
+from .exceptions import LtiError
+from .oauth import verify_oauth_body_signature
+
+
+log = logging.getLogger(__name__)
+
+
+def parse_grade_xml_body(body):
+    """
+    Parses values from the Outcome Service XML.
+
+    XML body should contain nsmap with namespace, that is specified in LTI specs.
+
+    Arguments:
+        body (str): XML Outcome Service request body
+
+    Returns:
+        tuple: imsx_messageIdentifier, sourcedId, score, action
+
+    Raises:
+        LtiError
+            if submitted score is outside the permitted range
+            if the XML is missing required entities
+            if there was a problem parsing the XML body
+    """
+    lti_spec_namespace = "http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0"
```

Comment:
This is the namespace for the XML document that the LTI outcome service endpoint expects to receive with requests to update component score. I don't see an issue with hard-coding this here.